### PR TITLE
docs(website): add views docs and typescript highlighter

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

File: packages/website/builder/config.json [hunk 1]
```diff
@@ -40,10 +40,25 @@
...
+    }, {
+      "name": "inferno",
+      "path": "../../node_modules/@cerebral/inferno/README"
+    }, {
+      "name": "preact",
+      "path": "../../node_modules/@cerebral/preact/README"
+    }, {
+      "name": "angularjs",
+      "path": "../../node_modules/@cerebral/angularjs/README"
+    }],
```
Comment (gaspard) [inline packages/website/builder/config.json]: We need to keep these sorted to help with navigation... Order is:

```
angularjs
inferno
preact
react
vue
```

File: packages/website/builder/config.json [hunk 2]
```diff
@@ -40,10 +40,25 @@
...
     "addons": [{
       "name": "router",
       "path": "../../node_modules/@cerebral/router/README"
```
Comment (gaspard) [inline packages/website/builder/config.json]: Could also sort these ones while we are here...

```
firebase
firebase-admin
forms
http
router
shortcuts
storage
useragent
```

File: packages/website/builder/config.json (hunk 1 above)
Comment (christianalfoni) [reply packages/website/builder/config.json]: Agreed, updated :)

File: packages/website/builder/config.json (hunk 2 above)
Comment (christianalfoni) [reply packages/website/builder/config.json]: Agreed, updated :) Removed `function-tree`, I think it deserves its own "thing". Not sure what, will think on it

**Comment by christianalfoni** [comment]
Okay, summary.

I changed to alphabetical order on views, but have not changed addons yet. I will make separate PR where I make everything alphabetical on API, also changing how collapsing/expanding works

**Pull request merged**